portfolio/src/main/webapp/index.html

```diff
@@ -12,15 +12,21 @@
     <div id="content">
         <h1>Kristen's Portfolio</h1>
         <div class="tab">
-            <button class="tablinks active" onclick="openTab(event,'home')">Home</button>
-            <button class="tablinks" onclick="openTab(event,'about-me')">About Me</button>
-            <button class="tablinks" onclick="openTab(event,'my-projects')">My Projects</button>
-            <button class="tablinks" onclick="openTab(event,'fun-facts')">Fun Facts</button>
-            <button class="tablinks" onclick="openTab(event,'travel-blog')">Travel Blog</button>
-            <button class="tablinks" onclick="openTab(event,'contact-me')">Contact Me</button>
+            <button class="tablinks active button" onclick="openTab(event,'home')">Home</button>
+            <button class="tablinks button" onclick="openTab(event,'about-me')">About Me</button>
+            <button class="tablinks button" onclick="openTab(event,'my-projects')">My Projects</button>
+            <button class="tablinks button" onclick="openTab(event,'fun-facts')">Fun Facts</button>
+            <button class="tablinks button" onclick="openTab(event,'travel-blog')">Travel Blog</button>
+            <button class="tablinks button" onclick="openTab(event,'contact-me')">Contact Me</button>
         </div>
         <div id="home" class="tabcontent">
             <h3>Hi there, and welcome to my portfolio! </h3>
+            <div id="graphic">
+              <img id="joke-pic" src="/images/graphic_design_is_my_passion.jpg" alt="A poorly photoshopped image that says \'Graphic design is my passion\' in Papyrus font">
+            </div>
+            <div id="hide-div">
+              <button id="show-button button" onclick="hideButton('hide-div');hideButton('graphic');fixSite()">Ahh, my eyes! Please fix this monstrosity of a website</button>
```

Comment:
its best to keep the inline javascript as small as possible. I would create a function in the js file that calls these three functions, then just call that function from the html

Reply:
Should be fixed after my last commit, it was definitely too crowded in that line!